code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/EdgePersonalizationResponseHandler.java

```diff
@@ -56,16 +57,25 @@ class EdgePersonalizationResponseHandler {
     private final MessagingCacheUtilities messagingCacheUtilities;
     private final ExtensionApi extensionApi;
     private final LaunchRulesEngine launchRulesEngine;
-    private final FeedRulesEngine feedRulesEngine;
+    private final ContentCardRulesEngine contentCardRulesEngine;
 
     private Map<Surface, List<Proposition>> propositions = new HashMap<>();
```

Comment:
We can rename this to `codeBasedPropositionsBySurface` to avoid confusion about its purpose and accidentally omitting attaching it in `retrieveMessages`